CITATION.cff

```diff
@@ -0,0 +1,132 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: UltraScan-III
+message: >-
+  A comprehensive data analysis framework for analytical
+  ultracentrifugation experiments, small angle scattering
+  experiments, and hydrodynamic bead modeling from atomic
+  structures
+type: software
+authors:
+  - given-names: Borries
+    family-names: Demeler
+    name-particle: Borries
```

Comment:
That lets your first name appear twice.
```
Borries Demeler, B., Brookes, E., E Gorbet, G., Savelyev, A., Mortezazadeh, S., & Dobler, L. UltraScan-III [Computer software]. https://github.com/ehb54/ultrascan3
```
```
@software{Borries_Demeler_UltraScan-III,
author = {Borries Demeler, Borries and Brookes, Emre and E Gorbet, Gary and Savelyev, Alexey and Mortezazadeh, Saeed and Dobler, Lukas},
license = {LGPL-3.0-or-later},
title = {{UltraScan-III}},
url = {https://github.com/ehb54/ultrascan3}
}
```
Removing this line should fix that.
```suggestion

```